Remotion/Web/Development.Analyzers/WxeResourcePageStepAnalyzer.cs

```diff
@@ -93,7 +93,7 @@ private void AnalyzeObjectCreationExpression (SyntaxNodeAnalysisContext ctx, Sym
 
       if (operation.Arguments[0].Value is not ITypeOfOperation typeOfOperation)
       {
-        if (SymbolEqualityComparer.Default.Equals(operation.Arguments[0].Parameter.Type, symbolContext.SystemTypeSymbol))
+        if (SymbolEqualityComparer.Default.Equals(operation.Arguments[0].Parameter!.Type, symbolContext.SystemTypeSymbol))
```

Comment:
I didn't look into this beyond the code I'm seeing. Typically, overriding nullability is a bad thing. It may be correct here. Would like a second opinion @a-ctor

Reply:
We should probably use `?.` instead as that won't fail and the equality still works in our favor.